Documentation/project-docs/profiling-api-status.md

```diff
@@ -43,4 +43,26 @@ The notes below will help you determine what profiling APIs are safe to use. The
 ##APIs definitely known not to work yet
 ###Windows
 ###Linux
+
+* ICorProfilerInfo:
```

Comment:
I'd prefer to leave it out for now, the better strategy is to just get tests up and running. Hopefully we can keep this list a short interim thing :)